packages/aws-lc-fips/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "aws-lc-fips"
```

Comment:
Ideally this package would be named `libcrypto` since it provides that shared library.

Reply:
I did name it this as we are also building libssl as per the comment below. I think for us, we do want libssl if possible, as we will link probably set up remote systemd-journal with TLS also.
We could split up the install packages though?